pkg/controller/plan/kubevirt.go

```diff
@@ -935,14 +936,27 @@ func (r *KubeVirt) GetGuestConversionPod(vm *plan.VMStatus) (pod *core.Pod, err
 	return
 }
 
-func (r *KubeVirt) UpdateVmByConvertedConfig(vm *plan.VMStatus, pod *core.Pod, step *plan.Step) (err error) {
+func (r *KubeVirt) getInspectionXml(pod *core.Pod) (string, error) {
+	inspectionUrl := fmt.Sprintf("http://%s:8080/inspection", pod.Status.PodIP)
+	resp, err := http.Get(inspectionUrl)
+	if err != nil {
+		return "", liberr.Wrap(err)
+	}
+	inspectionBytes, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", liberr.Wrap(err)
+	}
+	defer resp.Body.Close()
```

Comment:
I thnk the defer will be nicer before the 
`inspectionBytes, err := io.ReadAll(resp.Body)`
line, imidiatly after the` resp, err := http.Get(inspectionUrl)` error check